core/src/test/scala/unit/kafka/integration/UncleanLeaderElectionTest.scala

```diff
@@ -97,6 +98,15 @@ class UncleanLeaderElectionTest extends QuorumTestHarness {
     super.tearDown()
   }
 
+  override def kraftControllerConfigs(testInfo: TestInfo): Seq[Properties] = {
+    val properties = new Properties()
+    if (testInfo.getTestMethod.get().getName.contains("testUncleanLeaderElectionEnabled")) {
+      properties.setProperty("unclean.leader.election.enable", "true")
+    }
+    properties.setProperty("leader.imbalance.check.interval.seconds", "1")
```

Comment:
The `testTopicUncleanLeaderElectionEnableWithAlterTopicConfigs` is failing because we need to use the `unclean.leader.election.interval.ms` config now.